cogs/verification.py

```diff
@@ -0,0 +1,138 @@
+import aiohttp
+import json
+from random import choice, choices, randint, sample
+
+from discord import File, Error
+from discord.ext.commands import (
+    Cog,
+    Bot,
+    Context,
+    command,
+    CommandInvokeError,
+    has_permissions,
+)
+from discord.utils import get
+
+
+class Verification(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.config_full = json.loads(open("assets/config.json").read())
+        self.word_list_refresh_rate = 99
+        self.word_cache_size = 1000
+
+    @command()
+    @has_permissions(manage_guild=True)
+    async def verification(self, ctx: Context, state: bool) -> None:
+        """Enable or disable the verification system"""
+        config = self.config_full[str(ctx.message.guild.id)]
+
+        if state is True and config["verification_channel"] is None:
+            channel = await ctx.message.guild.create_text_channel(name="Verification")
+            role = await ctx.message.guild.create_role(name="Unverified")
+            config.update(verification_channel=channel.id, verification_role=role.id)
+            json.dump(
+                self.config_full,
+                open("config.json", "w"),
+                indent=2,
+                separators=(",", ": "),
+            )
+        elif state is False and config["verification_channel"] is not None:
+            channel = get(
+                ctx.message.guild.text_channels, id=config["verification_channel"]
+            )
+            role = get(ctx.message.guild.roles, id=config["verification_role"])
+            await channel.delete()
+            await role.delete()
+            config.update(verification_channel=None, verification_role=None)
+            json.dump(
+                self.config_full,
+                open("config.json", "w"),
+                indent=2,
+                separators=(",", ": "),
+            )
+
+    @command()
+    async def verify(self, ctx: Context) -> None:
+        """Verify yourself (the bot will DM you)"""
+        try:
+            self.verify.use_count += 1
+        except AttributeError:
+            self.verify.use_count = 1
```

Comment:
This seems irrelevant, keeping track of verification should be done on a per-server basis, not on bot-basis and will be done with database (possibly by API)